src/data/equiv/mul_add.lean

```diff
@@ -545,6 +561,10 @@ protected def mul_left₀ (a : G) (ha : a ≠ 0) : perm G :=
   left_inv := λ x, by { dsimp, rw [← mul_assoc, inv_mul_cancel ha, one_mul] },
   right_inv := λ x, by { dsimp, rw [← mul_assoc, mul_inv_cancel ha, one_mul] } }
 
+lemma _root_.group_with_zero.mul_left_bijective₀ (a : G) (ha : a ≠ 0) :
```

Comment:
Do we want this in the `group_with_zero` namespace? I think the `₀` suffix is good enough.